tiatoolbox/tools/pyramid.py

```diff
@@ -564,6 +566,12 @@ def get_tile(
                 The tile index in the x direction.
             y (int):
                 The tile index in the y direction.
+            res (int):
```

Comment:
@John-P Please can you update?

Reply:
@measty Please can you add the description?

Reply:
Addressed in this commit https://github.com/TissueImageAnalytics/tiatoolbox/pull/624/commits/97642299ef1b93c340133f73aa706c1af950b3b1